packages/go-discord-irc/metadata.nix

```diff
@@ -1,5 +1,5 @@
 {
   vendorSha256 = "1xfspsxsb9if3lc51fgcbxr0ab9127rdw3hgm8bimcd1cds0a4l2";
-  rev = "e6adabf26a60248e6a580551cebc2b0940979585";
+  rev = "e810d6b2109825a0e15f4cd01690846c1b75a147";
   sha256 = "1dn6m61rcii0jps1bhacahzkzx4mssgjxjw9hgyw5vw5gs1mj8wq";
```

Comment:
You need to update these shas too. Easiest way to do it is to nix-build this file and grab it from the errors.

Reply:
yeah goign to do over the weekend